kotlin/workflow-runtime/src/main/java/com/squareup/workflow/WorkflowHost.kt

```diff
@@ -137,8 +143,12 @@ interface WorkflowHost<in InputT, out OutputT : Any, out RenderingT> {
           }
       }
 
-    private fun <T> channelOf(value: T) = Channel<T>(capacity = 1)
-        .apply { offer(value) }
+    private fun <T> channelOf(value: T): () -> ReceiveChannel<T> {
+      return {
+        Channel<T>(capacity = 1)
+            .apply { offer(value) }
+      }
+    }
```

Comment:
Are `{ }` necessary here for return statement?

Reply:
What would be the alternative?

```kotlin
return fun(): ReceiveChannel<T> {
  Channel…
}
```

is just more verbose.

Reply:
My bad, misread the code :)